src/CSET/operators/plot.py

```diff
@@ -623,7 +641,7 @@ def plot_line_series(
 def plot_vertical_line_series(
     cube: iris.cube.Cube,
     filename: str = None,
-    series_coordinate: str = "pressure",
+    series_coordinate: str = "model_level_number",
```

Comment:
```suggestion
    series_coordinate: str = "pressure",
```
I do wonder whether `pressure` would be more common than `model_level_number`, and thus should be the default. Most recipes should override this however.

Reply:
As lfric won't have pressure unless it gets postprocessed for now we will more likely have model_level_number. So not committing this suggestion for now.